guides/source/command_line.md

```diff
@@ -124,6 +93,37 @@ development:
 
 It generated a database configuration corresponding to our choice of PostgreSQL.
 
+### Skipping Defaults
+
+If you wish to skip some files from being generated or skip some libraries
+entirely, you can pass one of the `--skip` arguments to the `rails new` command:
+
+```bash
+$ rails new sas --skip-active-storage
+Based on the specified options, the following options will also be activated:
+
+  --skip-action-mailbox [due to --skip-active-storage]
+  --skip-action-text [due to --skip-active-storage]
+
+      create
+      create  README.md
+      ...
+```
+
+In the above example, Action Mailbox and Action Text are skipped in addition to
+Active Storage because they depend on Active Storage functionality.
+
+For a full list of options (including what can be skipped), use `--help`:
+
+```bash
+$ rails new --help
+Usage:
+  rails new APP_PATH [options]
+
+Options:
+  ...
+```
+
```

Comment:
```suggestion
For a full list of options (including what can be skipped), use `rails new --help`.
```

Reply:
@zzak I suggested this change to make it stand out more and making it easier to coy the command 😄 
I'm ok with both.

It would be nice to have a copy button when hovering over single backtick commands.

Reply:
I also missed that you said "No need for the output", so maybe a happy middle ground would be just the command?
```suggestion
For a full list of options (including what can be skipped), use `--help`:

```bash
$ rails new --help
```
```